src/metaschema/oscal_mapping_metaschema.xml

```diff
@@ -0,0 +1,63 @@
+      <schema-name>OSCAL Control Mapping Model</schema-name>
+      <schema-version>1.0.3</schema-version>
+      <short-name>oscal-mapping</short-name>
+      <namespace>http://csrc.nist.gov/ns/oscal/1.0</namespace>
+      <json-base-uri>http://csrc.nist.gov/ns/oscal</json-base-uri>
+      <remarks>
+            <p>The OSCAL Control mapping format can be used to describe how a collection of security controls and related control enhancements relate to another collection of controls. The root of the Control Catalog format is <code>mapping-collection</code>.
+            </p>
+      </remarks>
+      <import href="oscal_metadata_metaschema.xml"/>
+      <import href="oscal_mapping-common_metaschema.xml"/>
+
+      <define-assembly name="mapping-collection">
+            <formal-name>Mapping Collection</formal-name>
+            <description>A collection of control mappings.</description>
+            <root-name>mapping-collection</root-name>
+            <define-flag name="uuid" as-type="uuid" required="yes">
+                  <formal-name>Mapping Collection Universally Unique Identifier</formal-name>
+                  <description>A globally unique identifier with cross-instance scope for this catalog instance. This UUID should be changed when this document is revised.</description>
+            </define-flag>
+
+            <model>
+                  <assembly ref="metadata" min-occurs="1"/>
+                  <assembly ref="mapping" min-occurs="1" max-occurs="unbounded">
+                        <group-as name="mappings"/>
+                  </assembly>
+                  <assembly ref="back-matter">
+                        <remarks>
+                              <p>Back matter including references and resources.</p>
+                        </remarks>
+                  </assembly>
+            </model>
+      </define-assembly>
+      <define-assembly name="mapping">
+            <formal-name>Control Mapping</formal-name>
+            <description>A mapping between two target resources.</description>
+            <define-flag name="uuid" as-type="uuid" required="yes">
+                  <formal-name>Mapping Universally Unique Identifier</formal-name>
+                  <description>A <a href="/concepts/identifier-use/#machine-oriented">machine-oriented</a>, <a href="/concepts/identifier-use/#globally-unique">globally unique</a> identifier with <a href="/concepts/identifier-use/#cross-instance">cross-instance</a> scope that can be used to reference this mapping definition elsewhere in this or other OSCAL instances. The locally defined <em>UUID</em> of the <code>mapping</code> can be used to reference the data item locally or globally (e.g., in an imported OSCAL instance). This UUID should be assigned <a href="/concepts/identifier-use/#consistency">per-subject</a>, which means it should be consistently used to identify the same subject across revisions of the document.</description>
```

Comment:
should the description above end with "consistently used to identify the same control mapping across revisions of the document"

Reply:
Indeed.

Reply:
Done in latest commit.